lib/avromatic/model/types/abstract_timestamp_type.rb

```diff
@@ -1,15 +1,22 @@
 # frozen_string_literal: true
 
+require 'active_support/time_with_zone'
+
 module Avromatic
   module Model
     module Types
-      class AbstractTimestampType
+      class AbstractTimestampType < AbstractType
         VALUE_CLASSES = [::Time].freeze
+        INPUT_CLASSES = [::Time, ::DateTime, ::ActiveSupport::TimeWithZone].freeze
 
         def value_classes
           VALUE_CLASSES
         end
 
+        def input_classes
+          INPUT_CLASSES
+        end
```

Comment:
Is there a pattern to when you define this as a method and when you set it as an instance variable in `initialize`? I'm wondering if you can standardize the approach (and if that's helpful).

Reply:
I set it in a variable in `initialize` if it's based on other attributes of the instance. I use a constant + reader method If it's static data. We could definitely standardize on the former but I don't see much benefit in that as long as the type classes abide by the same API contract. I'm happy to change it if you feel strongly about it though.

Reply:
I don't feel that strongly about it.